packages/node/src/client.test.ts

```diff
@@ -145,5 +145,114 @@ You'll find you API key at this location: https://app.bearer.sh/keys`
         expect(data).toEqual(okResponse)
       })
     })
+
+    describe('getAuth', () => {
+      const authId = 'test-auth-id'
+
+      const mockRequest = (data: any) => {
+        nock('https://auth.bearer.sh', {
+          reqheaders: {
+            authorization: secretKey,
+            ...headers
+          }
+        })
+          .get(`/apis/${integrationName}/auth/${authId}`)
+          .once()
+          .reply(200, data)
+      }
+
+      it('raises an error when there is no auth id set', async () => {
+        expect(api.getAuth()).rejects.toMatchSnapshot()
+      })
+
+      describe('OAuth 1', () => {
+        const details = {
+          callbackParams: { oauth_token: 'test-token', oauth_verifier: 'test-verifier' },
+          consumerKey: 'test-consumer-key',
+          consumerSecret: 'test-secret',
+          signatureMethod: 'HMAC-SHA1',
+          tokenSecret: 'test-token-secret',
+          accessToken: {
+            active: true,
+            value: 'test-token',
+            client_id: 'test-consumer-key',
+            iat: 1574087265,
+            token_type: 'oauth'
+          }
+        }
+
+        it('fetches the auth details and translates them to a friendly format', async () => {
+          mockRequest(details)
+
+          const authDetails = await api.auth(authId).getAuth()
+
+          expect(authDetails).toMatchSnapshot()
```

Comment:
```suggestion
          expect(authDetails).toMatchInlineSnapshot()
```
it facilitates reading expectations, what do you think?

Reply:
event shorter

```js
Object.assign({}, ...['whitelisted', 'keys'].map(key => ({ [key]: obj[key] })))
```

Reply:
> it facilitates reading expectations, what do you think?

I agree with using the inline version in general but these are quite big so I think using the separate file is better in this case.

Reply:
fair enough